study.md

```diff
@@ -34,7 +34,12 @@ including class. Why might Array redefine methods included from Enumerable?
 Please give reasons for the methods you list.
 
 ```md
-<!-- your answer here -->
+<!--
+any?
+zip
+I dont know why arrays would redefine methods included from enumerable
```

Comment:
https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=ruby+array+and+enumerable